tests/tacacs/test_ro_disk.py

```diff
@@ -67,6 +68,10 @@ def do_reboot(duthost, localhost, duthosts):
             localhost.wait_for(host=duthost.mgmt_ip, port=22, state="stopped", delay=5, timeout=60)
             rebooted = True
             break
+        except AnsibleConnectionFailure as e:
+            logger.error("DUT not reachable, exception: {} attempt:{}/{}".
+                         format(repr(e), i, retries))
+            wait(wait_time, msg="Wait {} seconds before retry.".format(wait_time))
```

Comment:
>wait [](http://example.com/codeflow?start=12&length=4)

Is it better to wait for both exception cases? If yes, you may move `wait` to outside `try-except` block.

Reply:
Fixed